docs/en/api/FS-API.md

```diff
@@ -187,23 +212,40 @@ Users can override the default policy class in the
 [configuration file]({{ '/en/operation/Configuration.html' | relativize_url }}) at property
 `alluxio.user.block.write.location.policy.class`. The built-in policies include:
 
-* **LocalFirstPolicy (alluxio.client.block.policy.LocalFirstPolicy)**
+* [LocalFirstPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/LocalFirstPolicy.html)
+
+  **This is the default policy.**
+
+  > A policy that returns local host first, and if the local worker doesn't have enough availability,
+  > it randomly picks a worker from the active workers list for each block write.
+
+* [LocalFirstAvoidEvictionPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/LocalFirstAvoidEvictionPolicy.html)
+
+  This is the same as `LocalFirstPolicy` with the following addition:
 
-    Returns the local worker first, and if it does not have enough capacity of a block,
-    randomly picks a worker from the active workers list. This is the default policy.
+  > If no worker meets the demands, return local host. USER_FILE_WRITE_AVOID_EVICTION_POLICY_RESERVED_BYTES
```

Comment:
ditto here about `local host`. i think it is more proper to use local worker

can you provide a bit more context on what `USER_FILE_WRITE_AVOID_EVICTION_POLICY_RESERVED_BYTES ` is? right now i'm reading it as a plain string; is it a configuration property? a metric key? also add backticks around it

also i'm just noticing, did this policy exist before? or did we just not document it?

Reply:
I don't know when it was added; I noticed it wasn't documented when I looked at the Javadoc so I added it here. I'll look into what the `USER_FILE_WRITE_AVOID_EVICTION_POLICY_RESERVED_BYTES` is and give a description underneath the quoted block.

Reply:
I updated the source-code's javadoc description of `getAvailableBytes()` based on what I could understand, but if someone could fact-check me on that that'd be much appreciated!